v03_pipeline/lib/tasks/write_imported_callset.py

```diff
@@ -122,27 +141,14 @@ def create_table(self) -> hl.MatrixTable:
         mt = select_relevant_fields(
             mt,
             self.dataset_type,
-            [
-                *(
-                    # info.AF is "required" if the relatedness check
-                    # is expected to run.
-                    ['info.AF']
-                    if (
-                        self.check_sex_and_relatedness
-                        and self.dataset_type.check_sex_and_relatedness
-                    )
-                    else []
-                ),
-                *(
-                    # this field is never required, the pipeline
-                    # will run smoothly even in its absence, but
-                    # will trigger special handling if it is present.
-                    ['info.CALIBRATION_SENSITIVITY']
-                    if hasattr(mt, 'info')
-                    and hasattr(mt.info, 'CALIBRATION_SENSITIVITY')
-                    else []
-                ),
-            ],
+            self.additional_row_fields(mt),
+        )
+        # This validation isn't override-able.  If a field is the wrong
+        # type, the pipeline will likely hard-fail downstream.
```

Comment:
or even worse, the pipeline will succeeed, data will update, and then search will totally break in seqr!